HACKING.md

```diff
@@ -119,7 +124,7 @@ You can run unit tests of the current checkout:
 
     make check
 
-These should finish very quickly and it is good practice to do it often.
+These should finish very quickly. It is a "good practice" to do this often.
```

Comment:
Why the quotes? This isn't meant in any ironic or indirect way.

Reply:
It's not meant to be ironic or indirect. "Good practice" is a saying.

We can drop it, though, if people would think it's ironic. It's more correct with the quotes, but if people think it's ironic, that's worse than being proper.